LayoutTests/imported/w3c/web-platform-tests/mediacapture-insertable-streams/MediaStreamTrackProcessor-stop.worker.js

```diff
@@ -0,0 +1,23 @@
+importScripts("/resources/testharness.js");
+
+function makeVideoFrame(timestamp) {
+  const canvas = new OffscreenCanvas(100, 100);
+  const ctx = canvas.getContext('2d');
+  return new VideoFrame(canvas, {timestamp});
+}
+
+promise_test(async t => {
+  const generator = new VideoTrackGenerator();
+  const processor = new MediaStreamTrackProcessor({track: generator.track});
+
+  const reader = processor.readable.getReader();
+  const writer = generator.writable.getWriter();
+
+  await new Promise(resolve => setTimeout(resolve, 500));
```

Comment:
can we have a comment on what we are waiting for here and how 500ms was chosen?

Reply:
Actually, we do not even need 500, we just need to execute in the next task, so 0 would be sufficient.